### DISCUSSION TITLE
I am getting package does not exist error.

### DISCUSSION THREAD

**Discussion by diyapkaba** [hyochan/react-native-iap#2710] [Q&A]
Hello Everyone,
I couldn't solve an issue for a week. Decided to text here for help. 

I followed documentation and checked it several times. Everything is correct. But i am still getting the following error for MainApplication.java 

> Task :app:compileAmazonDebugJavaWithJavac FAILED
C:\Users\OMEN\Repos\DsvlineApp\android\app\src\main\java\com\dsvline\dsvlineapp\MainApplication.java:12: error: package com.dooboolab.rniap does not exist
import com.dooboolab.rniap.RNIapPackage;
                          ^
C:\Users\OMEN\Repos\DsvlineApp\android\app\src\main\java\com\dsvline\dsvlineapp\MainApplication.java:30: error: cannot find symbol
          packages.add(new RNIapPackage());
                           ^
  symbol: class RNIapPackage
2 errors

Anyone to help please !

Best Regards.

**Comment by hyochan** [comment]
Thank you for using react-native-iap!

**Versions below v14 are no longer supported.** We cannot provide support or bug fixes for v12/v13 and earlier versions.

## Why Upgrade?

v14+ brings significant improvements:

- **Nitro Modules** - JSI-based native bridge for better performance
- **StoreKit 2** - Modern iOS in-app purchase API (iOS 15+)
- **Google Play Billing 7.0+** - Latest Android billing features
- **Unified API** - Cleaner, more consistent interface
- **Better error handling** - Unified `ErrorCode` enum across platforms
- **Active maintenance** - Bug fixes and new features

## Migration Guide

Please follow the migration guide to upgrade:

👉 **[Migration Guide (v13 to v14)](https://hyochan.github.io/react-native-iap/docs/guides/migration-v13-to-v14)**

## Requirements

- **iOS**: 15.0+
- **Xcode**: 16.4+
- **React Native**: 0.71+
- **Android**: Google Play Billing 7.0+

## Need Help?

If you encounter issues during migration, please:

1. Read the [Migration Guide](https://hyochan.github.io/react-native-iap/docs/guides/migration-v13-to-v14) thoroughly
2. Check the [Troubleshooting Guide](https://hyochan.github.io/react-native-iap/docs/guides/troubleshooting)
3. Open a new issue with v14+ if problems persist
